tests/test_s3.py

```diff
@@ -1726,71 +1727,99 @@ def test_enable_versioning(self):
 
     @functional
     def test_enable_logging(self):
+        bname = "superduper-and-magic"
+
         self.patch(s3.S3, "executor_factory", MainThreadExecutor)
+        # only augment with logging info to minimize API calls
         self.patch(
-            s3, "S3_AUGMENT_TABLE", [("get_bucket_logging", "Logging", None, None)]
+            s3,
+            "S3_AUGMENT_TABLE",
+            [("get_bucket_logging", "Logging", None, "LoggingEnabled")],
+        )
+        # and ignore any other buckets we might have in this test account
+        # to minimize the placebo data and API calls
+        self.patch(
```

Comment:
interesting technique, this is still a client side filter though so it still fetches all buckets from a data perspective. another technique that fetches the exact data is to use `source: config` with a `query` which will fetch the exact resource, the translation for s3 resources from config to describe format is a bit heavy though.

note this from a merge perspective this is fine, just fyi.